contrib/asciidoctor/template.py

```diff
@@ -0,0 +1,41 @@
+pkgname = "asciidoctor"
+pkgver = "2.0.20"
+pkgrel = 0
+makedepends = ["ruby"]
+depends = ["ruby"]
+pkgdesc = "Ruby-based AsciiDoc converter"
+maintainer = "Erica Z <zerica@callcc.eu>"
+license = "MIT"
+url = "https://asciidoctor.org"
+source = f"!https://rubygems.org/downloads/{pkgname}-{pkgver}.gem"
+sha256 = "835eabd445e4ae88f56a5f4e07593c3612b2be72eb661c612c3a8e1e17c57479"
+
+
+def do_install(self):
+    import os.path
+
+    gemdir = os.path.relpath(
+        self.do("ruby", "-e", "puts Gem.default_dir", capture_output=True)
+        .stdout.decode()
+        .strip(),
+        "/",
+    )
+
+    self.do(
```

Comment:
where does the gem come from? nothing copies it

Reply:
isn't it already in the builddir by default from download? still archived, but since it's the only file the cwd ends up having it in the right dir